services/metadata/backend/backend.go

```diff
@@ -19,28 +19,32 @@ import (
 	"github.com/snabb/sitemap"
 )
 
-// Representation of a project
 type Project struct {
 	ID          string      `json:"id"`
 	Name        string      `json:"name"`
 	Description string      `json:"description"`
 	Metadata    interface{} `json:"metadata"`
 }
 
+type Projects struct {
```

Comment:
I'm not sure what is happening here. I would prefer if you would extend `Project` to include another property that holds the json-ld.

Reply:
IMO best is to keep JSON-LD and JSON as separated collections, which I did here in `Projects`.